src/components/Routes.tsx

```diff
@@ -354,6 +354,7 @@ export function Routes() {
               {(dataFetcher) => {
                 return (
                   <LabwareDetails
+                    samplePositionResults={dataFetcher.visiumPermData.samplePositionResults}
```

Comment:
I think we need to make use of _samplePositions_ query on core side to fetch samplePosition information related to a labware. _samplePositionResults_ are not related to visiumPermData

Reply:
I updated the core code accordingly, so the samplePositions is part of the visimuPermData object, unless you mean this is not logically correct, and we need to perform two separate query

Reply:
Ohh, ok, In that case it is fine. But I couldn't see this update in src/graphql/queries/FindPermData.graphql.  I think that needs to updated as well

Reply:
I did not update the FindPerData.graphql file as it is referring to the LabwareFields type, which already been updated.